app/src/main/java/com/minuk/petcalendar/main/MainActivity.kt

```diff
@@ -0,0 +1,25 @@
+package com.minuk.petcalendar.main
+
+import androidx.navigation.fragment.NavHostFragment
+import androidx.navigation.ui.setupWithNavController
+
+import com.minuk.petcalendar.R
+import com.minuk.petcalendar.base.BaseActivity
+import com.minuk.petcalendar.databinding.ActivityMainBinding
+
+import dagger.hilt.android.AndroidEntryPoint
+
+@AndroidEntryPoint
+class MainActivity : BaseActivity<ActivityMainBinding>(R.layout.activity_main) {
+
+    override fun initLayout() {
+        initBottomNavigationView()
+    }
+
+    private fun initBottomNavigationView() {
+        val navHostFragment =
+            supportFragmentManager.findFragmentById(R.id.mainContainer) as NavHostFragment
```

Comment:
as?로 캐스팅 해서 안정성으로 챙겨주면 좋을 것 같아요
```suggestion
            supportFragmentManager.findFragmentById(R.id.mainContainer) as? NavHostFragment
```
밑에 setupWithNavController에서는 navHostFragment?.let {~~ } 식으로 적으면 좋을 것 같아요